scania-digital-design-system/tegel
# refactor(popover, tooltip): share code instead of duplicating

Moves shared logic of popover-menu, popover-canvas, and tooltip into a core component which they are all composed out of. 
This makes it so we wont have to fix bugs related to popovers in three separate places.

**How to test**  
1. Go to storybook
2. Check in the popover-canvas, popover-menu, and tooltip and verify everything works as expected.
3. Come up with your own method to verify if something does not work as expected.

**Suggested test steps**
